bin/exercism_local_tooling_webserver

```diff
@@ -27,7 +27,7 @@ post '/job' do
   logger.info `ls -lah #{input_dir}`
 
   # Run command
-  cmd = "/bin/sh bin/run.sh #{params[:exercise]} #{input_dir} #{output_dir}"
+  cmd = "./bin/run.sh #{params[:exercise]} #{input_dir} #{output_dir}"
```

Comment:
Do we know for sure that the `run.sh` is always executable in the test runners? Should we change the spec so that the entrypoint calls this directly then?

e.g. currently the entrypoins use bin/sh:
-  https://github.com/exercism/ruby-test-runner/blob/master/Dockerfile#L14
- https://github.com/exercism/javascript-test-runner/blob/master/Dockerfile#L35

Reply:
It should be - if not that would be a flaw with the test runners themselves, which they need to fix.  We can update it but it also shouldn't really matter if Docker uses /bin/sh to launch whatever shell is needed.

Reply:
My latest JS file already calls `bash`, though perhaps that specificity can be removed now.